Samples/TrendingMovies/TrendingMovies/AppDelegate.swift

```diff
@@ -0,0 +1,118 @@
+import Kingfisher
+import UIKit
+
+class AppDelegate: UIResponder, UIApplicationDelegate {
+    var window: UIWindow?
+    var tracer: Tracer?
+
+    func application(_: UIApplication, willFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]? = nil) -> Bool {
```

Comment:
`m`: I think we should add Sentry here with all features enabled. Basically https://github.com/getsentry/sentry-cocoa/blob/0b1660bd828e9e1b093de8dc250701364083eee5/Samples/iOS-Swift/iOS-Swift/AppDelegate.swift#L17-L30

Reply:
Added a note about this in the planning doc